libraries/azure_monitor_activity_log_alert.rb

```diff
@@ -32,7 +32,7 @@ def operations
 
   def scopes
     return unless exists?
-    properties&.scopes
+    properties&.scopes&.map { |scope| scope.delete_suffix('/') }
```

Comment:
Debating, whether it would be useful to have this after `super(opts, true)`
```
properties&.scopes&.map { |scope| scope.delete_suffix('/') }
```
This could be helpful for anyone that may access `scopes` via the full path rather than the method. If it was to be moved it may require a `!` to be added to update properties in place.

Reply:
Good catch!

Reply:
This is fixed now.